config/metasrv.example.toml

```diff
@@ -28,6 +28,13 @@ max_retry_times = 12
 # Initial retry delay of procedures, increases exponentially
 retry_delay = "500ms"
 
+# Failure detectors options.
+[failure_detector]
+threshold = 8.0
+min_std_deviation_millis = 100.0
```

Comment:
I think we should not use the `xxx_millis` such names for parameters. Can we use `humantime` just like other parameters for time settings? Can you create another PR to fix them? Thanks a lot. @tisonkun 

Please note such changes when doing the code review .@fengjiachun @MichaelScofield